software/preseq/lcextrap/main.nf

```diff
@@ -37,7 +37,7 @@ process PRESEQ_LCEXTRAP {
         $options.args \\
         $paired_end \\
         -output ${prefix}.ccurve.txt \\
-        $bam
+        -bam $bam
```

Comment:
```suggestion
        $bam
```

Reply:
We should keep this as it was before. The `-bam` parameter is actually coming from [here](https://github.com/nf-core/rnaseq/blob/3643a94411b65f42bce5357c5015603099556ad9/conf/modules.config#L118) and, hopefully this will allows developers to trick the module into accepting other sorts of input files without updating the module.